rust/makefile

```diff
@@ -26,8 +27,8 @@ help: makefile
 ## init: Install missing dependencies.
 .PHONY: init
 init:
-	rustup target add aarch64-apple-ios armv7-apple-ios armv7s-apple-ios x86_64-apple-ios i386-apple-ios
```

Comment:
Could you please split this line into the currently supported iOS targets (aarch64, x86_64) and the deprecated ones (skipping i386)? This would allow anyone with legacy plugins to still be able to fetch all targets from Rust 1.41.1